h/static/scripts/annotation-mapper.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+
+// Fetch the container object for the passed annotation from the threading
+// service, but only return it if it has an associated message.
+function getContainer(threading, annotation) {
+  var container = threading.idTable[annotation.id];
+  if (container === null || typeof container === 'undefined') {
```

Comment:
Ah, I see why you asked :) - I would recommend just using the falsey-ness of null/undefined etc. so that null/undefined are treated equivalently and also for consistency with logic that interfaces with third party code which may use either interchangeably.

Otherwise LGTM.